File: 9-palindrome-number/palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int n=x;
        int r=0;
        if(x<0){
            return false;
        }
        while(n>0){
            int d=n%10;
            r=r*10+d;
            n=n/10;
        }
       if(r==x){
        return true;
       } 
      else{
        return false;
      }
    }
}

File: 9-palindrome-number/palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        n=x
        rev=0
        if(x<0):
            return False
        while(n>0):
            d=n%10
            rev=rev*10+d
            n=n//10
        if(rev==x):
            return True
        else:
            return False        